contentcuration/contentcuration/frontend/RecommendedResourceCard/components/RecommendedResourceCard.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+
+  <KCard
+    title="sample title"
+    :headingLevel="2"
+    :to="sampleRoute"
+    layout="horizontal"
+    thumbnailScaleType="cover"
+    thumbnailDisplay="small"
+    :titleLines="3"
+    thumbnailSrc="https://domf5oio6qrcr.cloudfront.net/medialibrary/11525/0a5ae820-7051-4495-bcca-61bf02897472.jpg"
+  >
+    <template #aboveTitle>
+      <div>
+        <KIconButton
```

Comment:
[Required] Please change to `<KIcon>`. According to the designs it seems to me it doesn't need to be clickable. I think it will help to resolve one of the focus ring issues.

Reply:
Yes, its not clickable. Its more for information purposs.